zerver/tests/test_events.py

```diff
@@ -463,6 +464,29 @@ def test_change_realm_invite_required(self):
             error = schema_checker('events[0]', events[0])
             self.assert_on_error(error)
 
+    def test_change_realm_authentication_methods_enabled(self):
+        # type: () -> None
+        schema_checker = check_dict([
+            ('type', equals('realm')),
+            ('op', equals('update_dict')),
+            ('property', equals('default')),
+            ('data', check_dict([])),
+        ])
+        # Test every transition
+        for (auth_method) in \
```

Comment:
maybe name the `auth_method` variable `auth_method_dict` since it's not just one method?